python/ray/scripts/scripts.py

```diff
@@ -2003,6 +2004,37 @@ def global_gc(address):
     print("Triggered gc.collect() on all workers.")
 
 
+@cli.command(name="kuberay-autoscaler", hidden=True)
+@click.option(
+    "--cluster-name",
+    required=True,
+    type=str,
+    help="The name of the Ray Cluster.\n"
+    "Should coincide with the `metadata.name` of the RayCluster CR.",
+)
+@click.option(
+    "--cluster-namespace",
+    required=True,
+    type=str,
+    help="The Kubernetes namespace the Ray Cluster lives in.\n"
+    "Should coincide with the `metadata.namespace` of the RayCluster CR.",
+)
+@click.option(
```

Comment:
remove option?

Reply:
From what I understand, we may want to leave the possibility of using Redis to users who require it.
This is basically a private API and the Redis password is and optional option.

Reply:
Let me change the default to the empty string "", though for consistency with the rest of the call sites.

Also, let me add a warning to the help for this option.